README.md

```diff
@@ -23,6 +23,19 @@ Pomgen uses the lockfiles generated by [rules_jvm_external](https://github.com/b
 - pomgen < 2.0.0 is compatible with rules_jvm_external < 5.0
 - pomgen >= 2.0.0 is compatible with rules_jvm_external >= 5.0
 
+### Overidding Dependencies
+
+Set `override_file_paths` parameter in [general] section of .pomgenrc file.
+
+Value of `override_file_paths` should be comma separated paths containing .bzl files and having a mapping of an dep and overridden_dep.
+
+Example `overridden.bzl`
+
+overridden = {
+  "org.sfdc-bouncycastle:bc-fips": "@org_bouncycastle//:org_bouncycastle_bc_fips",
+  "org.sfdc-bouncycastle:bctls-fips": "@org_bouncycastle//:org_bouncycastle_bctls_fips"
+}
```

Comment:
I would probably enclose this in "```" e.g.

```
overridden = {
  "org.sfdc-bouncycastle:bc-fips": "@org_bouncycastle//:org_bouncycastle_bc_fips",
  "org.sfdc-bouncycastle:bctls-fips": "@org_bouncycastle//:org_bouncycastle_bctls_fips"
}
```